src/main.c

```diff
@@ -103,30 +113,66 @@ int load_conf_file(char *file_path)
     return 0;
 }
 
+static void run_test(int test_n) {
+    for (int i = 0; i < tests_exclude.size; i++) {
+        // If a match is found in the exclusion list, then we skip the test.
+        if (test_n == vector_get(&tests_exclude, i)) {
+            test_n = -1;
+            break;
+        }
+    }
+    // Skip the test if test_n is a negative number.
+    if (test_n >= 0) {
+        kernel_thunk_table[test_n]();
```

Comment:
`kernel_thunk_table` is not actually a kernel thunk table, right? It is an array of kernel API test functions, so perhaps it's clearer to update its name accordingly.

Reply:
Technically, you are correct. I think I did not rename the variable. And I may not know a better wording for it. Unless it's better to refer as... `kernel_api_tests` variable?

Reply:
Is this rename going to be part of this pull request, or in a later one?

Reply:
Later one, you'll know which one eventually. 😉

Reply:
resolved in #99 pull request.